data/books/programmation-de-droite-a-gauche-et-vice-versa.md

```diff
@@ -10,6 +10,6 @@ language: french
 isbn: "978-2-916466-05-7"
 links:
   - description: Order Online from Paracamplus
-    uri: https://paracamplus.com
+    uri: https://paracamplus.com - [1 Client error: Timeout was reached]
```

Comment:
I found the book at <https://www.decitre.fr/livres/programmation-de-droite-a-gauche-et-vice-versa-9782916466064.html> but not sure if the store is trustworthy. Any opinion?

Reply:
yes, this one is legit